<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerCLIModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerCLIModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>433</width>
        <height>397</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ctkCollapsibleButton" name="ModuleCollapsibleButton" native="true">
         <property name="text" stdset="0">
          <string>Module Title</string>
         </property>
         <layout class="QVBoxLayout" name="VerticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
            <item row="0" column="1">
             <widget class="qMRMLNodeComboBox" name="MRMLCommandLineModuleNodeSelector" native="true">
              <property name="nodeTypes" stdset="0">
               <stringlist notr="true">
                <string>vtkMRMLCommandLineModuleNode</string>
               </stringlist>
              </property>
              <property name="showHidden" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="ParameterSetLabel">
              <property name="text">
               <string>Parameter set:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="qSlicerCLIProgressBar" name="CLIProgressBar" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ButtonLayout">
     <item>
      <widget class="QPushButton" name="DefaultPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Reset parameters to default.</string>
       </property>
       <property name="text">
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkMenuButton" name="AutoRunPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>AutoRun</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Cancel the execution of the module</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Execute the module</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkMenuButton</class>
   <extends>QPushButton</extends>
   <header>ctkMenuButton.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerCLIProgressBar</class>
   <extends>QWidget</extends>
   <header>qSlicerCLIProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
</ui>
